<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>reveal.js</title>

		<link rel="stylesheet" href="reveal.js/css/reset.css">
		<link rel="stylesheet" href="reveal.js/css/reveal.css">
		<link rel="stylesheet" href="reveal.js/css/theme/moon.css">

		<!-- Theme used for syntax highlighting of code -->
		<link rel="stylesheet" href="reveal.js/lib/css/monokai.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				<section> 
					<h2> tranSMART for data analysis in Systems biology/immunology</h2>	 
					<img width="600" height="238" data-src="asset/transmart.png" alt="transmart logo">
					
				</section>
				<section>
				<section>
					<h3>tranSMART</h3>
						how to leverage multiple data type ?					
							<img width="500" height="400" data-src="asset/etricks_datatype.png" alt="transimmunome datatype">
							<p style="font-size:14px"> Gu et al, 2019 : Data and knowledge management in translational research: implementation of the eTRIKS platform for the IMI OncoTrack consortium</p>	
			
				</section>
				<section>
					<h3>tranSMART</h3>
					how to leverage multiple data type ?
					<p class="fragment"><img width="600" height="500" data-src="asset/tm_datatype.png" alt="transimmunome datatype"></p>
				</section>

				<section>
				 <h5> how to promote cooperation between professional with different background ? </h5>
					<ul class="fragment">
						<li>biologists/immunologists</li>
						<li>clinicians</li>
						<li>bioinformaticians</li>
						<li>Data managers</li>
					</ul>
				
				</section>
				</section>
				<section>
						<h3> What is tranSMART ?</h3>
						<p> a knowledge management plateform for biomedical research :</p>
						<ul>
							<li id="item" class="fragment" >a database </li>
						<img class="myImg" style="display:none;" width="500" height="400" data-src="asset/etricks_datatype.png" alt="transimmunome datatype">
							<li class="fragment">an ETL plateform</li>
							<li class="fragment">a graphical interface</li>
							<li class="fragment">a data analysis plateform (exploratory analysis)</li>		
						</ul>
						
				</section>
				<section>
				<section>
					<h3>Case report</h3>
					<h5>Data and knowledge management in translational research: implementation
						 of the eTRIKS platform for the IMI OncoTrack consortium</h5>
						 <p style="font-size:14px">Gu et al, 2019</p>
				</section>

				<section>
						<h3>Case report</h3>

				<p>developing an integrated knowledge management platform capable 
					of supporting the complexity of the data generated by biomedical research programmes.</p>
					
					<p style="font-size:14px"> Gu et al, 2019 </p>	
				</section>
			

				<section><h3>case report</h3>
					<p align="left">Methods  for  systematic  next  generation  oncology biomarker  development:</p>	
					<ul>
						<li>leverage multiple data type	</li>
						<li> promote cooperation  between differents institute 
							and between professional with different background</li>
					</ul>

				</section>
				<section><h3>case report</h3>
					
					<p align="left" >Two patient cohorts:</p>
					<ul>
						<li>prospectively : primary colon cancer surgery</li>
						<li>retrospectively : metastasis surgery</li>
					</ul>
										
				<p>build a colon cancer tissue  bank  containing  both  primary  and metastatic tumour samples, 
						together with associated normal tissues and biofluids.</p>	
				</section>
				<section>
					<h3>Imi OncoTrack types of data</h3>
					<p align="left">3 types of low dimensional data :</p>	
					<ul class="fragment">
						<li>the observational clinical cohorts,</li>
						<li>the drug response data from the cell-line models</li>
						<li>the drug response data from the xenograft models</li>
					</ul> 
					<p align="left" class="fragment">The high dimensional omics data:</p>
					<ul class="fragment">
						<li>Variants among germline, primary and metastatic tumour material	</li>
						<li>Confirmatory genomic analyses of xenograft and cell cultures	</li>
						<li>Quantification of RNA transcripts from clinical and preclinical samples	</li>
						<li>Quantification of small non-coding RNA (miRNA)		</li>
							<li>Analysis of DNA Methylation		</li>
					</ul>
				</section>
			
				
				<section><img width="700" height="600" data-src="asset/cohorte.png" alt="imt OncoTrack sample organisation"></section>
					<section><img width="700" height="700" data-src="asset/etricks_datatype.png" alt="transimmunome datatype">	</section>
					<section><img width="700" height="700" data-src="asset/transmartuioncotrack.png" alt="transimmunome datatype">	</section>

				</section>
					<section>
						<h3>data analysis with transmart</h3>
						<p align="left">Analysis tools for data exploration and hypothesis generation:</p>
						<ul>
							<li>Summary Statistics</li>
							<li>Advanced workflow</li>
							<li>SmartR</li>
							<li>Fractalis</li>
						</ul>
					</section>
					<section>
						<h3>SmartR</h3>
						<p>exploratory statistic and data visualisation plugin</p>
						<p>dynamic and interactive visualisation </p>
					</section>
					<section><img width="700" height="600" data-src="asset/SmartR.jpeg" alt="SmartR">
					</section>
					<section>
						<img width="700" height="600" data-src="asset/SmartR_architecture.jpeg" alt="SmartR">

					</section>
					<section>
						<h3>Fractalis</h3>
						plateform independant tool for data exploration 
					</section>
					<section>
						<img width="700" height="600" data-src="asset/fractalis.jpeg" alt="fractalis">

					</section>

					<section>
						 <h3>key concept</h3> 
						<ul>
							<li> data format agnostic : microETL</li>
							<li>allow parrallelization on multiple node</li>
							<li>plug every analysis script in python or R</li>
							<li>relay analysis requests to frameworks such as Hail that can analyze large-scale genotype data</li>
							<li>good development practices</li>
						</ul>
					</section>
					<section><h3>workflow developpment strategy</h3>
					<p align="left">2 methods</p>
					<ul>
						<li>script pluged on fractalis without knowledge of fractalis. </li>
						<li>script using celery interface to parrallelize traitment.</li>
					</ul>
					</section>
					<section>
						<h4>miR-1269 promotes metastasis and forms a positive feedback loop with TGF-β.</h4>
							<p style="font-size: 14px;" >Bu P. et all</p>
					</section>
					<section>
							<img width="700" height="600" data-src="asset/fractalis_fig3.jpeg" alt="fractalis">

					</section>
				</section>
				<section>
					<h3> Where we are now ?</h3>
					<ul class="fragment">
						<li>tranSMART is functional on our server and contain the data from the 167 first patients </li>
						<li> installation and loading was done by Rancho Bio Science</li>
					</ul >
					<h6 class="fragment">Objective</h6>
					<ul class="fragment">
						<li>do without the transmart plateform installed by Rancho Bio Science.</li>
						<li>be autonomous for dataloading in transmart.</li>
						<li>load Repertoire Sequencing data in tranSMART</li>
						<li> connect tranSMART and iReceptor+</li>
					</ul>
				</section>
				<section>
					<h3>sources</h3>
					<p>Data and knowledge management in translational research: implementation of the eTRIKS platform for the IMI OncoTrack consortium </p>
					
					<p> Fractalis: a scalable open-source service for platform-independent interactive visual analysis of biomedical data
							Sascha Herzinger, Valentin Grouès, Wei Gu, Venkata Satagopam, Peter Banda, Christophe Trefois, Reinhard Schneider</p>
					
					<p>Developing Interactive Plug-ins for tranSMART Using the SmarR Framework: The Case of Survival Analysis. Christian Knell, Martin Sedlmayr, Jan Christoph</p>
					
					<p> miR-1269 promotes metastasis and forms a positive feedback loop with TGF-β.
						Bu P1, Wang L2, Chen KY3, Rakhilin N3, Sun J4, Closa A5, Tung KL2, King S2, 
						Kristine Varanko A2, Xu Y2, Huan Chen J6, Zessin AS7, Shealy J3, 
						Cummings B8, Hsu D7, Lipkin SM9, Moreno V5, Gümüş ZH10, Shen X11.</p>

				</section>
				<section>
					<p>
					As patient survival drops precipitously from early-stage cancers to late-stage and metastatic 
					cancers, microRNAs that promote relapse and metastasis can serve as prognostic and predictive 
					markers as well as therapeutic targets for chemoprevention. Here we show that miR-1269a promotes 
					colorectal cancer (CRC) metastasis and forms a positive feedback loop with TGF-β signaling. miR-1269a 
					is upregulated in late-stage CRCs, and long-term monitoring of 100 stage II CRC patients revealed that
					 miR-1269a expression in their surgically removed primary tumors is strongly associated with risk of CRC relapse 
					and metastasis. Consistent with clinical observations, miR-1269a significantly increases the ability of 
					CRC cells to invade and metastasize in vivo. TGF-β activates miR-1269 via Sox4, while miR-1269a enhances 
					TGF-β signaling by targeting Smad7 and HOXD10, hence forming a positive feedback loop. Our 	findings 
					suggest that miR-1269a is a potential marker to inform adjuvant chemotherapy decisions for CRC patients and a potential therapeutic target to deter metastasis.
					</p>
				</section>
				<section>
					<p>The post‐genomic era promises to pave the way to a personalized understanding of 
						disease processes, with technological and analytical advances helping to solve some
						 of the world's health challenges. Despite extraordinary progress in our understanding 
						of cancer pathogenesis, the disease remains one of the world's major medical problems.
						New therapies and diagnostic procedures to guide their clinical application are urgently 
						required. OncoTrack, a consortium between industry and academia, supported by the Innovative
						 Medicines Initiative, signifies a new era in personalized medicine, which synthesizes current technological advances in omics techniques, systems biology approaches, and mathematical modeling. A truly personalized molecular imprint of the tumor micro‐environment and subsequent diagnostic and therapeutic insight is gained, with the ultimate goal of matching the “right” patient to the “right” drug and identifying predictive biomarkers for clinical application. This comprehensive mapping of the colon cancer molecular landscape in tandem with crucial, clinical functional annotation for systems biology analysis provides unprecedented insight and predictive power for colon cancer management. Overall, we show that major biotechnological developments in tandem with changes in clinical thinking have laid the foundations for the OncoTrack approach and the future clinical application
						 of a truly personalized approach to colon cancer theranostics.</p>
				</section>
			</div>
		</div>

		<script src="reveal.js/js/reveal.js"></script>
		<script src="js/custom.js"></script>
		<script type="text/javascript">
			// More info about config & dependencies:
			// - https://github.com/hakimel/reveal.js#configuration
			// - https://github.com/hakimel/reveal.js#dependencies
			Reveal.initialize({
				dependencies: [
					{ src: 'reveal.js/plugin/markdown/marked.js' },
					{ src: 'reveal.js/plugin/markdown/markdown.js' },
					{ src: 'reveal.js/plugin/notes/notes.js', async: true },
					{ src: 'reveal.js/plugin/highlight/highlight.js', async: true }
				]
			});
		</script>
	</body>
</html>
